Remotion/Data/DomainObjects/Persistence/Rdbms/DbCommandBuilders/Specifications/UpdatedColumnsSpecification.cs

```diff
@@ -56,8 +56,9 @@ public void AppendColumnValueAssignments (StringBuilder statement, IDbCommand db
       var columnsWithParameters = _columnValues.Select(
           cv =>
           {
-            var parameter = cv.Column.StorageTypeInfo.CreateDataParameter(dbCommand, cv.Value);
-            parameter.ParameterName = sqlDialect.GetParameterName(cv.Column.Name);
+            var storageTypeInfo = cv.Column.StorageTypeInfo;
+            var parameterName = sqlDialect.GetParameterName(cv.Column.Name);
+            var parameter = sqlDialect.CreateDataParameter(dbCommand, storageTypeInfo, parameterName, cv.Value);
```

Comment:
The usage pattern of `CreateDataParameter` is always the same. We start with a column, get the `storageTypeInfo` and `parameterName` and pass it to `CreateDataParameter`. We might want to consider if it makes more sense to pass the column into `CreateDataParameter` instead. What do you think?

Reply:
I get what you mean, but I honestly can't see which is cleaner. The current pattern is more testable though. @martin-herwich?